server/api/room.ts

```diff
@@ -350,37 +352,37 @@ const addToQueue: RequestHandler<
 	OttResponseBody<unknown>,
 	OttApiRequestAddToQueue
 > = async (req, res) => {
+	const body = OttApiRequestAddToQueueSchema.parse(req.body);
 	let points = 5;
-	if ("videos" in req.body) {
-		points = 3 * req.body.videos.length;
+	if ("videos" in body) {
+		points = 3 * body.videos.length;
 	}
 	if (!(await consumeRateLimitPoints(res, req.ip, points))) {
 		return;
 	}
 	const room = (await roommanager.getRoom(req.params.name)).unwrap();
 
-	let roomRequest: AddRequest;
-	if ("videos" in req.body) {
+	let roomRequest: AddRequest | undefined;
```

Comment:
I would prefer this to remain as just `AddRequest`. Do we need to add undefined?

Reply:
No, that's my bad. I just forgot to delete that.